tests/e2e/cucumber/features/smoke/fullTextSearch.ocis.feature

```diff
@@ -0,0 +1,89 @@
+Feature: Search
+  As a user
+  I want to do full text search
+  So that I can find the files with the content I am looking for
+
+  Background:
```

Comment:
Is there going to be more scenarios in this file that needs this whole background?

Reply:
No for now.